src/Compilers/Helpers/SimpleAnalyzerAssemblyLoader.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Loads analyzer assemblies from their original locations in the file system.
+    /// Assemblies will only be loaded from the locations specified when the loader
```

Comment:
@mavasani I don't think that's quite right, actually.

According to http://blogs.msdn.com/b/abhinaba/archive/2014/02/18/net-ngen-explicit-loads-and-load-context-promotion.aspx, only the Load context supports NGEN images. Sometimes LoadFrom is "promoted" to Load, and you'll get the NGEN image loaded, but in general LoadFrom does not prefer native images.